mona_sdk/client.py

```diff
@@ -801,6 +831,12 @@ def get_insights(
             },
         )
 
+        return (
+            app_server_response
```

Comment:
why don't we wrap this again with handle_service_error? 
Here and I believe in a few other places if the returned value from app server is the expected return structure we just return as is, while if we do this then if anyone changes the endpoint there it will affect the SDK which isn't desired. We should extract the error message from the response and rewrap it I believe (across the board, if other such cases exist)